pkgpanda/docs/filesystem_layout.md

```diff
@@ -3,7 +3,7 @@ pkgpanda packages expect a file system organized as described here so that symli
 may be generated with the bootstrap tarball before actually being placed on the host system.
 
 ```bash
-/etc/mesosphere/roles/{master,slave,aws,aws_master}
+/etc/mesosphere/roles/{master,slave,slave_public,aws,aws_master}
```

Comment:
These AWS roles should go, they're AWS-specific and aren't helpful for a general doc. If we wanted to be complete we'd need to add Azure roles too. But if we include all the roles for all the deployment methods, this line will just be a nuisance to maintain and will probably become out of date. We should stick to the roles that apply in all deployments: `master`, `slave`, and `slave_public`.